makefiles/testing.mak

```diff
@@ -1,7 +1,7 @@
 # -*- makefile -*-
 .PHONY: check units fuzz noise tmain tinst clean-units clean-tmain clean-gcov run-gcov codecheck cppcheck dicts cspell
 
-check: tmain units
+check: tmain units roundtrip
```

Comment:
That's the tricky part: it's useful to have it so roundtrip actually checks something everywhere, but it doesn't work on Windows.  So maybe leave it out until Windows support is fixed?

Reply:
I will also attack this crazy issue. If I cannot find a solution, too, I will do as you write and I will wait for @k-takata's help.

Reply:
I have read your efforts for making roundtrip work on MSYS2. Thank you very much.
I know well how it is painful.